extensions/amp-iframe/0.1/amp-iframe.js

```diff
@@ -664,12 +669,13 @@ export class AmpIframe extends AMP.BaseElement {
    * @private
    */
   updateSize_(height, width) {
-    if (!this.isResizable_) {
+    if (!this.isResizable_ && !this.hasErroredEmbedSize_) {
```

Comment:
This needs to be nested inside the `isResizable_` if statement. As is, this would allow the second request to succeed.

Reply:
good catch!